packages/salesforcedx-vscode-apex-replay-debugger/src/telemetry/telemetry.ts

```diff
@@ -56,4 +56,32 @@ export class TelemetryService {
       });
     }
   }
+
+  public sendLaunchEvent(logSizeStr: string, errorMsg: string): void {
+    if (this.reporter !== undefined && this.isTelemetryEnabled) {
+      this.reporter.sendTelemetryEvent('launchDebuggerSession', {
+        extensionName: EXTENSION_NAME,
+        logSize: logSizeStr,
+        errorMessage: errorMsg
+      });
+    }
+  }
+
+  public sendCheckpointEvent(errorMsg: string): void {
+    if (this.reporter !== undefined && this.isTelemetryEnabled) {
+      this.reporter.sendTelemetryEvent('updateCheckpoints', {
+        extensionName: EXTENSION_NAME,
+        errorMessage: errorMsg
+      });
+    }
+  }
+
+  public sendErrorEvent(errorMsg: string): void {
+    if (this.reporter !== undefined && this.isTelemetryEnabled) {
+      this.reporter.sendTelemetryEvent('error', {
+        extensionName: EXTENSION_NAME,
+        errorMessage: errorMsg
```

Comment:
@lcampos - Do you have a preference on how you want to log errors in AppInsights? I'm sending an error message in an error event. We can also have a callstack, but it's in JS source.

Reply:
This format is ok since it'll allow us to filter by and build graphs using extensionName, errorMessage, logSize, etc.